src/index.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+// Selects the HTML Elements the events will occur on
+const increaseTempButton = document.querySelector('#increaseTempControl');
+const decreaseTempButton = document.querySelector('#decreaseTempControl');
+const tempValue = document.querySelector('#tempValue');
+const landscape = document.querySelector('#landscape');
+const headerCityName = document.querySelector('#headerCityName');
+const currentTempButton = document.querySelector('#currentTempButton');
+const cityNameInput = document.querySelector('#cityNameInput');
```

Comment:
Consider collecting these event listeners into a callback function, like `registerEventHandlers`, which will then be accessible once the DOM has completely loaded. Something like:
```js
const registerEventHandlers = () => {
    // your elements and event listeners here
};
document.addEventListener('DOMContentLoaded', registerEventHandlers);
```
This will ensure the HTML elements are loaded and event handlers find the appropriate HTML tags they need.